webapp/tests/test_routes.py

```diff
@@ -23,3 +33,104 @@ def test_if_post_and_form_data_then_extract_correct_form_data(self, app):
             update_data, extracted_form_data = extract_information_from_request()
 
         assert extracted_form_data == ImmutableMultiDict(form_data)
+
+
+@pytest.fixture
+def configuration_with_production_environment_testing_route_policy():
+    in_production_value = Config.ALLOW_TESTING_ROUTES
+    Config.ALLOW_TESTING_ROUTES = ProductionConfig.ALLOW_TESTING_ROUTES
+
+    yield Config
+
+    Config.ALLOW_TESTING_ROUTES = in_production_value
+
+
+@pytest.fixture
+def configuration_with_staging_environment_testing_route_policy():
+    in_production_value = Config.ALLOW_TESTING_ROUTES
+    Config.ALLOW_TESTING_ROUTES = StagingConfig.ALLOW_TESTING_ROUTES
```

Comment:
If you follow my comment above, then this should be a functional env. (Which makes more sense than staging anyway, since that's where we need this route.)